app/views/table_on_steroids/_table_on_steroids.html.erb

```diff
@@ -70,16 +71,14 @@
   <table class="table table-striped table-bordered table-hover">
     <thead>
       <tr>
+        <%= hidden_field_tag('knowledge[order]', params&.dig(:knowledge,:order), id: "knowledge_order#{table_on_steroid_id}", form: "knowledge_base_filters#{table_on_steroid_id}") %>
+        <%= hidden_field_tag('knowledge[ascending]', params&.dig(:knowledge,:ascending), id: "knowledge_ascending#{table_on_steroid_id}", form: "knowledge_base_filters#{table_on_steroid_id}") %>
         <% columns.each_pair do | k , v | %>
           <th class="column column-<%= k %> <%= ' d-none ' unless selected_columns.include?(k) %> <%= v[:css_class].join(' ') if(v[:css_class]) %> <%= table_on_steroid_id %>">
+            <%= v[:label] %>
             <% if v[:type] == 'order' %>
-              <label>
-                <%= v[:label] %>
-                <%= radio_button('knowledge', 'order', k, onchange: "$('#knowledge_base_filters#{table_on_steroid_id}').submit();", checked: params[:knowledge] && params[:knowledge][:order] == k, form: "knowledge_base_filters#{table_on_steroid_id}" ) %>
-                v
-              </label>
-            <% else %>
-              <%= v[:label] %>
+              <%= image_tag "up_arrow_white.svg", :style => "margin-left:5px;margin-right:5px;margin-bottom:5px;height:8px;cursor:pointer", onclick: "$('#knowledge_order#{table_on_steroid_id}').val('" + k + "');$('#knowledge_ascending#{table_on_steroid_id}').val('true');$('#knowledge_base_filters#{table_on_steroid_id}').submit();" %>
```

Comment:
Any way to make these styles non-inline?

Reply:
👍